src/components/buttonSend/buttonSend.scss

```
@@ -0,0 +1,10 @@
+.button-send {
+  box-sizing: border-box;
+  height: 40px;
+  width: 40px;
+  border-radius: 50%;
+}
+
+.sendable {
+display: none;
```

надо поправить форматирование
```suggestion
  display: none;
```